assets/prefabs/magicStaff.prefab

```diff
@@ -7,5 +7,8 @@
         "icon" : "LightAndShadowResources:magicStaff",
         "usage": "IN_DIRECTION"
     },
-    "RaycastOnActivate": {}
+    "Range": {
+        "value": 30.0
```

Comment:
This is an absolutely arbitrary number, I guess we can tweak it later on if we find out that it is too big or too small.